sql/numbertype.go

```diff
@@ -86,6 +86,70 @@ func MustCreateNumberType(baseType query.Type) NumberType {
 	return nt
 }
 
+func NumericZeroValue(t Type) interface{} {
+	nt := t.(numberTypeImpl)
+	switch nt.baseType {
+	case sqltypes.Int8:
+		return int8(0)
+	case sqltypes.Uint8:
+		return uint8(0)
+	case sqltypes.Int16:
+		return int16(0)
+	case sqltypes.Uint16:
+		return uint16(0)
+	case sqltypes.Int24:
+		return int32(0)
+	case sqltypes.Uint24:
+		return uint32(0)
+	case sqltypes.Int32:
+		return int32(0)
+	case sqltypes.Uint32:
+		return uint32(0)
+	case sqltypes.Int64:
+		return int64(0)
+	case sqltypes.Uint64:
+		return uint64(0)
+	case sqltypes.Float32:
+		return float32(0)
+	case sqltypes.Float64:
+		return float64(0)
+	default:
+		panic(fmt.Sprintf("%v is not a valid number base type", nt.baseType.String()))
+	}
+}
+
+func NumericUnaryValue(t Type) interface{} {
```

Comment:
Just get the zero and add one, no need for this method

Reply:
`Type.Zero() + 1` is a type mismatch